src/datatypes/oppfolgingTypes.ts

```diff
@@ -0,0 +1,59 @@
+export interface Me {
+    erBruker: boolean;
+    erVeileder: boolean;
+    id: string;
+}
+
+export interface OppfolgingStatus {
+    fnr: string;
+    aktorId: string;
+    veilederId: string;
+    reservasjonKRR: boolean;
+    kanVarsles: boolean;
+    manuell: boolean;
+    underOppfolging: boolean;
+    underKvp: boolean;
+
+    gjeldendeEskaleringsvarsel: Eskaleringsvarsel;
+    kanStarteOppfolging: boolean;
+
+    oppfolgingsPerioder: OppfolgingsPeriode;
+    harSkriveTilgang: boolean;
+    inaktivIArena: boolean;
+    kanReaktiveres: boolean;
+    inaktiveringsdato: Date;
+    erSykmeldtMedArbeidsgiver: boolean;
+    servicegruppe: string;
+    formidlingsgruppe: string;
+    rettighetsgruppe: string;
+}
+
+export interface OppfolgingsPeriode {
+    string: string;
+    begrunnelse: string;
+    kvpPerioder?: Array<KvpPeriode>;
```

Comment:
Her kan man også skrive 
```suggestion
    kvpPerioder?: KvpPeriode[];
```